ui/util/lbryURI.js

```diff
@@ -157,7 +160,14 @@ function parseURIModifier(modSeperator: ?string, modValue: ?string) {
   }
 
   if (claimId && (claimId.length > claimIdMaxLength || !claimId.match(/^[0-9a-f]+$/))) {
-    throw new Error(__(`Invalid claim ID %claimId%.`, { claimId }));
+    const hashIndex = claimId.indexOf('#');
+
+    if (hashIndex) {
+      pathHash = claimId.substring(hashIndex, claimId.length);
+      claimId = claimId.substring(0, hashIndex);
+    } else {
+      throw new Error(__(`Invalid claim ID %claimId%.`, { claimId }));
+    }
```

Comment:
- [ ] :exclamation: `hashIndex` will always be true because a negative number is 'truthy' in javascript.
- [ ] :arrow_down: `pathHash = claimId.substring(hashIndex, claimId.length);` -- can omit the second parameter if grabbing until the end. Shorter code.
- [ ] :bulb: As a pre-caution to catch future odd urls coming in, validate the new `claimId` length and characters again after stripping off the `pathHash`